code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -180,7 +174,7 @@ Auto Patrol: []"},
 		return
 	if(iscarbon(A))
 		var/mob/living/carbon/C = A
-		if(!C.stunned || arrest_type)
+		if(!C.stunned)

```

Comment:
Please keep the detain function, remove it from the interface & topic calls, allow mappers/admins to use it if they wish.